Modules/Office365DSC/DSCResources/MSFT_SPOSiteAuditSettings/MSFT_SPOSiteAuditSettings.psm1

```diff
@@ -0,0 +1,180 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param (
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $SiteUrl,
```

Comment:
SPOSite is using the parameter Url. Isn't it a good idea to use a consistent parameter name?

Reply:
Done